packages/experiments/src/components/Button/Button.tsx

```diff
@@ -1,10 +1,13 @@
 import { createComponent } from '../../Foundation';
 import { useButtonState as state } from './Button.state';
-import { ButtonStyles as styles, ButtonTokens as tokens } from './Button.styles';
+import { ButtonClassNames, ButtonStyles as styles, ButtonTokens as tokens } from './Button.styles';
 import { IButtonProps } from './Button.types';
 import { ButtonView } from './Button.view';
 
+const classNames = Object.values(ButtonClassNames);
+
 export const Button: React.StatelessComponent<IButtonProps> = createComponent(ButtonView, {
+  classNames,
```

Comment:
Any reason why `ButtonView` is passed in as arg 0, instead of part of the options?

Reply:
Yes, because it brings the API of `createComponent` to parity with styled components as part of the [evolution of `styled`](https://github.com/OfficeDev/office-ui-fabric-react/issues/8823). You always need a view, but not necessarily any of the options, including styles.